tributors/utils/prompt.py

```diff
@@ -27,3 +27,32 @@ def request_input():
             lines.append(line)
 
     return "\n".join(lines)
+
+
+def choice_prompt(prompt, choices, choice_prefix=None, multiple=False):
+    """Ask the user for a prompt, and only return when one of the requested
+       options is provided.
+       Parameters
+       ==========
+       prompt: the prompt to ask the user
+       choices: a list of choices that are valid.   
+       multiple: allow multiple responses (separated by spaces)
```

Comment:
what style of docstring formatting is it?  doesn't remind me original `pydoc` (?) and not quite right for `numpy` (no types, no empty line before Parameters)

Reply:
Vanessa style :)